metals/src/main/scala/scala/meta/internal/metals/ReferenceProvider.scala

```diff
@@ -113,24 +113,6 @@ final class ReferenceProvider(
           info.symbol.owner,
           Descriptor.Type(info.displayName)
         )
-    // Returns true if `info` is a synthetic `copy` or `apply` of the occurrence class symbol.
-    def isCopyOrApplyMethod(info: SymbolInformation): Boolean =
```

Comment:
@olafurpg I removed it since it didn't seem intuitive to find apply/copy for case class symbols. I can change that to be behind a flag - I am not sure what was the motivations for adding them to references in the first place :thinking:

Reply:
The motivation was mostly for testing purposes, to preserve a bijection between definition/references.

I’m fine with removing this behavior but it may require adjustments in ReferenceLspSuite

Reply:
Fixed the tests in ReferenceLspSuite, should be alright.